### ISSUE TITLE
set_cookie_store called without save_cookie_store won't persist to disk

### ISSUE THREAD

**Issue by nicholaswild** [gstark/tesla-api#1]
I believe with HTTPClient calling set_cookie_store without calling save_cookie_store won't persist the store to disk. You also don't need to cal set_cookie_store at all if you just want to treat the cookies as volatile. Perhaps this was just in preparation for supporting more options in this area.

Great work!

**Comment by gstark** [comment]
Yes, I was experimenting with the cookie store but I wasn't sure how to make it work. I think the API should work to try to use any existing cookie and if it fails to authenticate to re-login and obtain new session cookies.

If you are interested, please send a pull request with tests and I'd be happy to merge.
